src/google/protobuf/compiler/cpp/cpp_message_field.cc

```diff
@@ -972,9 +972,6 @@ GenerateInlineAccessorDefinitions(io::Printer* printer) const {
       "  // @@protoc_insertion_point(field_add:$full_name$)\n"
       "  return $name$_.Add();\n"
       "}\n");
-  }
-
-  if (!dependent_field_) {
```

Comment:
This section has been changed dramatically in the internal branch. And this part no longer exist in the other repo. Could you revert this? It will make the conflict resolve easier.